src/redux/authSlice.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from "@reduxjs/toolkit";
+
+const getUser = () => {
+  const existingUser = localStorage?.getItem("user");
+  if (existingUser) {
+    return JSON.parse(existingUser);
```

Comment:
Could you please see the changes?  The project crashes the complete project after some time.
![Screenshot from 2023-09-04 12-42-01](https://github.com/sumon766/frontend-doctor-appointment/assets/86189460/81161f2d-5cfe-430a-bff0-cf704d139a1d)

When you refresh, Nothing appears
![Screenshot from 2023-09-04 14-26-41](https://github.com/sumon766/frontend-doctor-appointment/assets/86189460/a9482f34-f4e4-4f28-9229-cb0260608442)